modules/guardduty/main.tf

```diff
@@ -1,4 +1,8 @@
 resource "aws_guardduty_detector" "default" {
+
+  # checkov check not relevant to us. Want it set for all
+  #checkov:skip=CKV2_AWS_3: "Ensure GuardDuty is enabled to specific org/region"
+
```

Comment:
```suggestion

  # checkov check not relevant to us. Want it set for all
  #checkov:skip=CKV2_AWS_3: "Ensure GuardDuty is enabled to specific org/region - This will be applied in each account the baseline will run in and in the all supported regions."

```